schema/data/prod/form_json/production.json

```diff
@@ -48,9 +52,22 @@
                 "required": ["productEmissions"]
               }
             ]
+          },
+          "isProductionAmountOptional" : {
+            "oneOf": [{
+              "properties": {
+                "isProductionAmountOptional": { "const": true }
```

Comment:
As mentioned above, can you rename this to `requiresProductAmount`
It's not that it's optional, it's that it shouldn't appear in the form.
While we're at it, can you rename `quantity` to `productAmount`?
If the `productAmount` is not required, there shouldn't be any `productUnits` either